pkg/services/ngalert/store/database.go

```diff
@@ -17,6 +17,7 @@ const AlertDefinitionMaxTitleLength = 190
 // AlertingStore is the database interface used by the Alertmanager service.
 type AlertingStore interface {
 	GetLatestAlertmanagerConfiguration(*models.GetLatestAlertmanagerConfigurationQuery) error
+	GetAllLatestAlertmanagerConfiguration() ([]*models.AlertConfiguration, error)
```

Comment:
I think we should start adding `context.Context` here @gotjosh?

Reply:
Yes, I remember an announcement several days ago. I will add.